web/src/lib/utils/asset-utils.ts

```diff
@@ -361,17 +365,19 @@ export const stackAssets = async (assets: AssetResponseDto[]) => {
   parent.stack ??= [];
   parent.stack = parent.stack.concat(children, grandChildren);
   parent.stackCount = parent.stack.length + 1;
-
-  notificationController.show({
+  const notificationObject = {
```

Comment:
What are we trying to achieve here? The `button` property is also not a public property AFAIK